B/Binutils/build_tarballs.jl

```diff
@@ -1,93 +1,69 @@
-    "e56673694db395e25b31808b4fbb9a7005e6875f",
-    "https://github.com/tpoechtrager/cctools-port.git" =>
-    "ecb84d757b6f011543504967193375305ffa3b2f",
-    "./bundled",
+    ArchiveSource("https://ftp.gnu.org/gnu/binutils/binutils-$(version.major).$(version.minor).tar.xz", "645c25f563b8adc0a81dbd6a41cffbf4d37083a382e02d5d3df4f65c09516d00"),
+    DirectorySource("$(@__DIR__)/bundled"),
 ]
 
-# Bash recipe for building across all platforms
-script = """
-# FreeBSD build system for binutils apparently requires that uname sit in /usr/bin/
-ln -sf \$(which uname) /usr/bin/uname
-
-# On MacOS, we don't actually install Binutils.  We install libtapi and cctools.  Le sigh.
-if [[ $(compiler_target) == *apple* ]]; then
-    mkdir -p \${WORKSPACE}/srcdir/apple-libtapi/build
-    cd \${WORKSPACE}/srcdir/apple-libtapi/build
-
-    # Install libtapi
-    cmake ../src/apple-llvm/src \\
-        -DLLVM_INCLUDE_TESTS=OFF \\
-        -DCMAKE_BUILD_TYPE=RELEASE \\
-        -DCMAKE_INSTALL_PREFIX=\${prefix}
-    make -j\${nproc} VERBOSE=1
-    make install
-
-    # Install cctools.  Someday, try this without disabling the clang assembler!
-    cd \${WORKSPACE}/srcdir/cctools-port/cctools
-    ./configure --prefix=\${prefix} \\
-        --target=$(compiler_target) \\
-        --host=\${MACHTYPE} \\
-        --with-libtapi=\${prefix}
-    make -j\${nproc}
-    make install
-else
-    cd \${WORKSPACE}/srcdir/binutils-*/
+script = raw"""
+cd ${WORKSPACE}/srcdir/binutils-*/
 
-    #atomic_patch -p1 "\${WORKSPACE}/srcdir/patches/binutils_COFF_bfd.patch"
-    update_configure_scripts
+./configure --prefix=${prefix} \
+    --target=${target} \
+    --build=${MACHTYPE} \
+    --host=${target} \
+    --disable-dependency-tracking \
+    --enable-deterministic-archives \
+    --disable-werror \
+    --disable-gprof \
+    --disable-gprofng \
+    --disable-gas \
+    --disable-gold \
+    --disable-ld \
+    --enable-install-libbfd \
+    --enable-install-libctf \
+    --enable-install-libiberty \
+    --enable-plugins \
+    --enable-targets=${target} \
+    --disable-nls \
+    --enable-64-bit-bfd \
+    --disable-static \
+    --enable-shared
 
-    ./configure --prefix=\${prefix} \\
-        --target=$(compiler_target) \\
-        --host=\${MACHTYPE} \\
-        --with-sysroot="\${prefix}/$(compiler_target)/sys-root" \\
-        --enable-multilib \\
-        --disable-werror
-
-    make -j\${nproc}
-    make install
-fi
-
-# Finally, create a bunch of symlinks stripping out the target so that
-# things like `nm` "just work", as long as we've got our path set properly.
-for f in \${prefix}/bin/$(compiler_target)-*; do
-    fbase=\$(basename \$f)
-    ln -s \$fbase \${prefix}/bin/\${fbase#$(compiler_target)-}
-done
+make -j${nproc}
+make install
 """
 
-# These are the platforms we will build for by default, unless further
-# platforms are passed in on the command line
-platforms = [Platform("x86_64", "linux")]
+platforms = supported_platforms(; exclude=!Sys.islinux)
```

Comment:
```suggestion
platforms = supported_platforms(; experimental=true)
```

Reply:
`experimental=true` is no-op nowadays, can just remove it.